common/service/config/config.go

```diff
@@ -271,7 +271,7 @@ type (
 		Enabled                bool  `yaml:"enabled"`
 		InitialFailoverVersion int64 `yaml:"initialFailoverVersion"`
 		// RPCName indicate the remote service name
-		RPCName string `yaml:"rpcName"`
+		RPCName RPCName `yaml:"rpcName"`
```

Comment:
I thought you wanted to make a default value here, correct?

Reply:
I am trying to put in the default value while doing the unmarshalling, I guess this is what I had hinted on. 
I do agree on you comment on enums, do you have any particular approach which tackles both of these things?

Reply:
I thought it can be something like 
````
RPCName string `yaml:"rpcName", default:"true`
```

Reply:
@longquanzheng I don't think this is a valid syntax for defining the default value
I guess there is no construct of providing defaults in go-yaml

Reply:
Yes you are right. There is some third party library can do that but I don't think we want to use. 
I think we just need to check if the string is empty, and if it is then set a default value to it.

Reply:
Yes, this is what seems right.